mail_tracking/models/mail_message.py

```diff
@@ -63,50 +70,30 @@ def tracking_status(self):
             partners -= partners_already
             for partner in partners:
                 # If there is partners not included, then status is 'unknown'
+                # Because can be an Cc recipinet
+                isCc = False
+                if partner.email in email_cc_list:
+                    email_cc_list.discard(partner.email)
+                    isCc = True
                 partner_trackings.append((
-                    'unknown', False, partner.name, partner.id, partner.email))
+                    'unknown', False, partner.name, partner.id, isCc))
+            for email in email_cc_list:
+                # If there is Cc without partner
+                partner_trackings.append((
+                    'unknown', False, email, False, True))
```

Comment:
I don't know if this is something you can actually alter, but if you can, would it make sense to use a list of `dict()`s instead? If all is positional and we have so many positions, it gets confusing, don't you think?